llama_index/vector_stores/mongodb.py

```diff
@@ -65,7 +60,17 @@ def __init__(
         insert_kwargs: Optional[Dict] = None,
         **kwargs: Any,
     ) -> None:
-        """Initialize params."""
+        """Args:
+        mongodb_client: A MongoDB client.
```

Comment:
I'd suggest "Initialize the vector store", and then format the `Args:` block consistent with the other functions.